src/items/equipment/bows/bronze-crossbow.js

```diff
@@ -1,11 +1,6 @@
-import Equipment from "../equipment.js";
-import { EQUIPMENT } from "../../constants/constants.js";
-
-export default class BronzeCrossbow extends Equipment {
-    // Attack types
-    slot = EQUIPMENT.SLOTS.WEAPON;
-    skill = EQUIPMENT.WEAPON_TYPES.RANGED;
+import Bow from "./bow.js";
 
+export default class BronzeCrossbow extends Bow {
```

Comment:
Might want a crossbow class. They do use bolts instead of arrows so the animation would be different. Idk if we even care that much about crossbows though

Reply:
Done https://github.com/EthanVieira/rs-clicker/pull/61/commits/f56516055f88cc3579f2713c3aac1ed45cda1cb1